### PR TITLE
Create RFC233

### PR DESCRIPTION
**Pull request by alabamenhu**

### CODE REVIEW

File: 20th/articles/rfc223.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    @foobar = @foo + @bar;          # (12); wait what?  Recall that arrays in numeric context are the number of elements, or 6 here.
+    
+Your instinctive reaction might be to say that we'd want to end up with `(0,22,44,66,88,110)` which is simple enough to handle in a basic `map` or `for` loop (using the zip operator, Raku can do this simply as `@foo Z+ @bar`).
+But remember what a superposition means: anything done happens to all the values, so each value in @foo needs to be handle with each value in @bar, which requires at least two loops if done via `map` or `for` (the cross operator in Raku can do this simply as `@foo X+ @bar`).
+We actually want `(0, 2, 4, 6, 8, 10, 20, 22, 24, 26, 28, 30, 40, 42, 44, 46, 48, 50 … )`.  More difficult, then, would be to somehow compare this value:
+
+    @foobar > 10;
+    
+There is no map method we can attach to `@foobar` to check its values against 10, we'd need to instead map the `> 10` into `@foobar`.
+But by using superpositioning, we can painless do all of the above with a single use of `map`, `for, or anything else that generates linenoise:
```
Comment (Altai-man) [inline 20th/articles/rfc223.md:51]: Missed the closing `.

**Review by Altai-man — CHANGES_REQUESTED** [review]

**Review by JJ — APPROVED** [review]
It's already uploaded, so well...

File: 20th/articles/rfc223.md (hunk 1 above)
Comment (JJ) [reply 20th/articles/rfc223.md:51]: This was probably fixed when uploading it, so thanks.

Comment (alabamenhu) [reply 20th/articles/rfc223.md:51]: Indeed I did fix it.  @Altai-man ++

**Pull request merged**